connector/datadogconnector/go.mod

```diff
@@ -100,7 +100,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/system/socket v0.59.0 // indirect
 	github.com/DataDog/datadog-agent/pkg/util/winutil v0.59.0 // indirect
 	github.com/DataDog/datadog-agent/pkg/version v0.59.0 // indirect
-	github.com/DataDog/datadog-api-client-go/v2 v2.31.0 // indirect
+	github.com/DataDog/datadog-api-client-go/v2 v2.33.0 // indirect
```

Comment:
hmmm looks like https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36339 did not work as I expected

Reply:
reverted for now, I'll look closer at how to configure renovate bot